e2e/pages/app.po.ts

```diff
@@ -0,0 +1,46 @@
+import {browser, by, element, protractor} from 'protractor';
+
+export class CloudstackUiPage {
+
+  navigateTo() {
+    return browser.get('/');
+  }
+
+  getParagraphText() {
+    return element(by.css('app-root h1')).getText();
+  }
+
+  clickButtonbyText(text: string) {
+    element(by.buttonText(text)).click();
+  }
+
+  clickButtonbyClass(classname: string) {
+    element(by.css('.' + classname)).click();
+  }
+
+  waitUrlContains (expected: string) {
+    return browser.wait(protractor.ExpectedConditions.urlContains(browser.baseUrl + '/' + expected), 5000);
+  }
+
+  checkButtonIsNotClickable(nameButton: string) {
```

Comment:
move to  buttonIsClickable and verify the result in test

Reply:
Fixed. This method not used in e2e-tests for login/logout, but it can be used in future tests, so I saved it.